### ISSUE TITLE
Support for Plasma color schemes

### ISSUE THREAD

**Issue by openmindead** [PapirusDevelopmentTeam/materia-kde#66]
What do you think about making Materia Plasma themes follow KDE color schemes instead of hard-coded ones? I'm talking not about Kvantum themes, but about Plasma themes. Breeze, for instance, acquires accent color from any other color scheme and the whole desktop and apps look consistent. This also will look good when using Materia Plasma theme + Breeze widget style for apps, which many people prefer (with their own color of choice).

**Comment by varlesh** [comment]
Kvantum not support this, it's senseless

**Comment by openmindead** [comment]
При чём тут Квантум? Я говорю о темах Плазмы, чтобы они реагировали на смену цветовой схемы. А у тебя гвоздями синий хайлайт/селекшн приколочен. Вот о чём речь.

**Comment by varlesh** [comment]
@openmindead Вообще идея этого пака была как унификация Qt/GTK при минимум усилий со стороны пользователя. Устанавливаешь тему GTK, устанавливаешь пак и всё - пользуйся.
Какой смысл рисовать хайлайт с цветовой схемы, если оно залочено во всех остальных виджетак квантума и GTK?
Я сам не одобряю этот фиолетовый цвет в темной теме, но разрабу GTK глубоко плевать на мое мнение. У него все по феншую и Material HIG

**Comment by varlesh** [comment]
Попробую портировать Materia Mono... Если получится добиться более менее нормального результата, то выложу в master

**Comment by openmindead** [comment]
@varlesh Да, я понимаю, и, кстати, спасибо, что этот проект вообще существует.
Но тут есть три момента:
- темой плазмы можно пользоваться и без Квантума
- GTK перекрашивают через oomox, например
- наконец, если уж браться за дело и перекрашивать темы Kvantum, то было бы замечательно, если бы не требовалось ещё и Plasma врукопашную править. Я каждый раз после перекройки твоей темы (Kvantum) под манжаровские цвета Breath вздыхаю и оставляю плазму на "когда-нибудь потом" :)

**Comment by varlesh** [comment]
Тут вот баг нарисовался:
https://bugs.kde.org/show_bug.cgi?id=422043
Решил запилить отдельные цвета для панели и виджетов и фиг там((

**Comment by varlesh** [comment]
Preview:
![image](https://user-images.githubusercontent.com/8083855/82811475-ea892080-9ea1-11ea-97f0-0a2ccf57afe7.png)

**Comment by openmindead** [comment]
> Тут вот баг нарисовался:
> https://bugs.kde.org/show_bug.cgi?id=422043
> Решил запилить отдельные цвета для панели и виджетов и фиг там((

Будем наблюдать, что тут скажешь. Посмотрим, что Нэйт скажет.

Превью прикольное, мне кажется, многим чего-то такого давно не хватало. Похоже на Breeze, но с вменяемыми иконками, и это хорошо)

**Comment by varlesh** [comment]
начальная поддержка запилена
проблема в том, что ColorScheme-ViewBackground не  работает.
Будем посмотреть... а пока так
![image](https://user-images.githubusercontent.com/8083855/82828538-53809080-9ec2-11ea-9658-7e7ffcf26837.png)

**Comment by varlesh** [comment]
@notmart @Montel @aleixpol Sorry guys, for disturb you, but maybe you see why `ViewBackground` use colors from `ButtonBackground` on plasma Theme:
https://bugs.kde.org/show_bug.cgi?id=422043

**Comment by varlesh** [comment]
![82828538-53809080-9ec2-11ea-9658-7e7ffcf26837](https://user-images.githubusercontent.com/8083855/82829476-6a27e700-9ec4-11ea-9219-5d1f96e90773.png)

**Comment by varlesh** [comment]
```
<style id="current-color-scheme" type="text/css">
   .ColorScheme-Background {
color:#181818; 
} 
.ColorScheme-ButtonBackground { 
color:#343434; 
} 
.ColorScheme-Highlight { 
color:#1a73e8; 
} 
.ColorScheme-Text { 
color:#dfdfdf; 
} 
.ColorScheme-ViewBackground { 
color:#242424; 
} 
.ColorScheme-NegativeText { 
color:#ff4747; 
}
  </style>
```
